### PR TITLE
Add simple dynamic routes for future CooperTS docs

### PR DESCRIPTION
**Pull request by edmondthui**
Created dynamic routing based on document id or did, showing Doc: {did} on the page.

Documentation: https://nextjs.org/docs/routing/dynamic-routes

### CODE REVIEW

File: docs/pages/index.tsx [hunk 1]
```diff
@@ -16,10 +20,12 @@ const Home: NextPage = () => {
         </h1>
 
         <div className={styles.grid}>
-          <a href="#" className={styles.card}>
-            <h2>Documentation &rarr;</h2>
-            <p>In-depth information on CooperTS features and API</p>
-          </a>
+          <Link href="doc/1">
```
Comment (pjrebsch) [inline docs/pages/index.tsx:23]: What is `1`?

File: docs/pages/index.tsx [hunk 2]
```diff
@@ -1,8 +1,12 @@
 import type { NextPage } from 'next';
 import Head from 'next/head';
+import { useRouter } from 'next/router';
+import Link from 'next/link'
 import styles from '../styles/Home.module.css';
 
 const Home: NextPage = () => {
+  const router = useRouter();
```
Comment (pjrebsch) [inline docs/pages/index.tsx:8]: why does this need to be here?

File: docs/pages/index.tsx (hunk 1 above)
Comment (edmondthui) [reply docs/pages/index.tsx:23]: 1 is just a placeholder, it is a dynamic route to the 1 doc

File: docs/pages/index.tsx (hunk 2 above)
Comment (edmondthui) [reply docs/pages/index.tsx:8]: This is how nextJS does routes. The router is and object that handles navigation

https://nextjs.org/docs/api-reference/next/router#router-object

Comment (edmondthui) [reply docs/pages/index.tsx:8]: I can remove this since we're not using it.

**Review by pjrebsch — APPROVED** [review]

**Pull request merged**